## 02-addindexpages.patch by Christian Welzel <gawain@camlann.de>
##
## DP: add some missing html pages to protect against directory listing

Index: typo3-dummy-4.3.0/fileadmin/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/fileadmin/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/fileadmin/_temp_/index.html
===================================================================
--- typo3-dummy-4.3.0.orig/fileadmin/_temp_/index.html	2010-05-04 21:27:46.000000000 +0200
+++ typo3-dummy-4.3.0/fileadmin/_temp_/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -4,4 +4,4 @@
 	<TITLE></TITLE>
 <META http-equiv=Refresh Content="0; Url=/">
 </HEAD>
-</HTML>
\ No newline at end of file
+</HTML>
Index: typo3-dummy-4.3.0/fileadmin/user_upload/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/fileadmin/user_upload/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/fileadmin/user_upload/_temp_/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/fileadmin/user_upload/_temp_/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/typo3conf/ext/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/typo3conf/ext/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/typo3conf/l10n/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/typo3conf/l10n/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/typo3temp/cs/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/typo3temp/cs/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/typo3temp/GB/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/typo3temp/GB/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/typo3temp/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/typo3temp/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/typo3temp/llxml/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/typo3temp/llxml/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/typo3temp/pics/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/typo3temp/pics/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/typo3temp/temp/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/typo3temp/temp/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/uploads/index.html
===================================================================
--- typo3-dummy-4.3.0.orig/uploads/index.html	2010-05-04 21:27:46.000000000 +0200
+++ typo3-dummy-4.3.0/uploads/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -4,4 +4,4 @@
 	<TITLE></TITLE>
 <META http-equiv=Refresh Content="0; Url=../">
 </HEAD>
-</HTML>
\ No newline at end of file
+</HTML>
Index: typo3-dummy-4.3.0/uploads/media/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/uploads/media/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/uploads/pics/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/uploads/pics/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
Index: typo3-dummy-4.3.0/uploads/tf/index.html
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ typo3-dummy-4.3.0/uploads/tf/index.html	2010-05-04 23:26:06.000000000 +0200
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+	<TITLE></TITLE>
+<META http-equiv=Refresh Content="0; Url=/">
+</HEAD>
+</HTML>
